docs/deploy/index.md

```diff
@@ -101,25 +58,8 @@ For development:
 $ minikube addons disable ingress
 ```
 
-2. Use the [docker daemon](https://github.com/kubernetes/minikube/blob/master/docs/reusing_the_docker_daemon.md)
-3. [Build the image](../development.md)
-4. Perform [Mandatory commands](#mandatory-commands)
-5. Install the `nginx-ingress-controller` deployment [without RBAC roles](#install-without-rbac-roles) or [with RBAC roles](#install-with-rbac-roles)
-6. Edit the `nginx-ingress-controller` deployment to use your custom image. Local images can be seen by performing `docker images`.
-
-```console
-$ kubectl edit deployment nginx-ingress-controller -n ingress-nginx
-```
-
-edit the following section:
-
-```yaml
-image: <IMAGE-NAME>:<TAG>
-imagePullPolicy: IfNotPresent
-name: nginx-ingress-controller
-```
-
-7. Confirm the `nginx-ingress-controller` deployment exists:
+2. Execute `make dev-env`
```

Comment:
Currently we perform the following:
```
kubectl set image \
    deployments \
    --namespace ingress-nginx \
    --selector app=ingress-nginx \
    nginx-ingress-controller=${REGISTRY}/nginx-ingress-controller:${TAG}
```

but in the future minikube will already have this deployment in the kube-system namespace and once that happens we can just edit the deployment on the kube-system namespace instead of creating a new one.

Reply:
I don't want to assume minikube is only way to run tests